docs/README.md

```diff
@@ -3,100 +3,103 @@
-    - [framework-provider-azure-\*](#framework-provider-azure-)
-  - [Configuration and environments](#configuration-and-environments)
-    - [Booster configuration](#booster-configuration)
-    - [Environments](#environments)
-  - [Extending Booster with Rockets!](#extending-booster-with-rockets)
-    - [Naming recommendations](#naming-recommendations)
-    - [Infrastructure extensions](#infrastructure-extensions)
-    - [Runtime extensions (Not yet implemented)](#runtime-extensions-not-yet-implemented)
-    - [Deploy and init hooks (Not yet implemented)](#deploy-and-init-hooks-not-yet-implemented)
+  * [Contributing](#contributing)
+  * [Framework Core](#framework-core)
+  * [Framework Types](#framework-types)
+  * [Framework integration tests](#framework-integration-tests)
+  * [Providers](#providers)
+    + [framework-provider-aws-\*](#framework-provider-aws-)
+    + [framework-provider-local-\*](#framework-provider-local-)
+    + [framework-provider-kubernetes-\*](#framework-provider-kubernetes-)
+    + [framework-provider-azure-\*](#framework-provider-azure-)
+  * [Configuration and environments](#configuration-and-environments)
+    + [Booster configuration](#booster-configuration)
+    + [Environments](#environments)
+  * [Extending Booster with Rockets!](#extending-booster-with-rockets)
+    + [Naming recommendations](#naming-recommendations)
+    + [Infrastructure extensions](#infrastructure-extensions)
+    + [Runtime extensions (Not yet implemented)](#runtime-extensions-not-yet-implemented)
+    + [Deploy and init hooks (Not yet implemented)](#deploy-and-init-hooks-not-yet-implemented)
 - [Debugging and testing Booster applications](#debugging-and-testing-booster-applications)
-  - [Running Booster applications locally](#running-booster-applications-locally)
-    - [Local development prerequisites](#local-development-prerequisites)
-    - [Starting your application](#starting-your-application)
-    - [Performing Auth requests](#performing-auth-requests)
-    - [Performing GraphQL requests](#performing-graphql-requests)
-    - [Inspect Database information](#inspect-database-information)
-  - [Testing Booster applications](#testing-booster-applications)
-  - [Booster examples](#booster-examples)
+  * [Running Booster applications locally](#running-booster-applications-locally)
+    + [Local development prerequisites](#local-development-prerequisites)
+    + [Starting your application](#starting-your-application)
+    + [Performing Auth requests](#performing-auth-requests)
+    + [Performing GraphQL requests](#performing-graphql-requests)
+    + [Inspect Database information](#inspect-database-information)
+  * [Testing Booster applications](#testing-booster-applications)
+    + [Testing with sinon-chai](#testing-with-sinon-chai)
+    + [Recommended files](#recommended-files)
+  * [Booster examples](#booster-examples)
 - [Frequently Asked Questions](#frequently-asked-questions)
 
 <!-- tocstop -->
```

Comment:
I believe this shouldn't be committed? 😅

Reply:
But those are docs as they should be, after running `update-tocs` command. Documentation wasn't changed, but it is not formatted as it should be... 😥

Reply:
Gah! I have to create another GH Action for this 😂 

Thanks for this!